.gitignore

```diff
@@ -3,3 +3,5 @@ spec/dummy
 .bundle
 
 *.log
+
+.project
```

Comment:
Don't worry. All you need to do is:

1. edit `.gitignore` and remove this change
2. `git add .gitignore`
3. `git commit --amend --no-edit` -> add new changes to last commit and don't change the commit message
4. `git push --force`